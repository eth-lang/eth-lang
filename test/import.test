; vim: set ft=clojure:

; simple named import
(import http)

; aliased imports
(import util u)

; destructed imports
(import helpers (derp herp))
