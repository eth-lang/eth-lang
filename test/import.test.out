http = require("http");
u = require("util");
derp = require("helpers").derp;
herp = require("helpers").herp;

